src/database/migrations/20190901120620-adding_to_user.js

```diff
@@ -26,7 +26,10 @@ export default {
         'Developer',
         'QA',
         'Maintenance',
+        'User',
```

Comment:
We should try as much as possible to not edit an existing migration. You should move these changes to a new migration file